src/frontend/src/pages/AdminToolsPage/FinanceConfig/CreateVendorModal.tsx

```diff
@@ -36,10 +36,9 @@ const CreateVendorModal = ({ showModal, handleClose }: NewVendorProps) => {
   const {
     handleSubmit,
     control,
-    formState: { isValid },
-    reset
+    reset,
+    formState: { errors }
   } = useForm({
-    mode: 'onChange',
     resolver: yupResolver(schema),
```

Comment:
can you update the schema in this file so that the error name is 'Vendor Name is Required' instead of 'Account Name is Required'